ivy/functional/backends/torch/linear_algebra.py

```diff
@@ -509,10 +509,15 @@ def vector_norm(
 vector_norm.support_native_out = True
 
 
+
```

Comment:
you should only keep two empty lines between each chunk of code, you can imitate other similar files for the correct format.